src/Pages/Tv/Tv.js

```diff
@@ -965,84 +980,94 @@ const UpdateWatchSec = async (data) => {
   //   }
   // };
 
+  // To prevent multi-tap misfires
+  let lastTapTime = 0;
 
-// To prevent multi-tap misfires
-let lastTapTime = 0;
+  const handleTap = (e) => {
```

Comment:
```suggestion
const handleTap = useCallback(debounce((e) => {
  if (energy.current <= 5) return;
```

try this to memoizes